source/develop/release-management/features/network/lldp.html.md

```diff
@@ -0,0 +1,277 @@
+Example of relevant configuration options are:
+
+* The VLAN configuration of the host has to match the VLAN configuration of the
+  switch port. Other way roud: If a host should be connected to a given VLAN,
+  there might be only a single network interface of the host which is connected
+  to a switch port which allows the usage of this VLAN.
+
+* According to section "5. Switch Configuration" in [2], the bonding modes
+  balance-rr (0), balance-xor (2), broadcast (3), 802.3ad (4) may require
+  configuration on switch side.
+
+* The switch port might have constraints about the MTU.
+
+
+The host will not expose information by LLDP to the switch.
+
+## Prerequisites
+
+LLDP has to be enabled on the switch connected to the relevant oVirt host.
+
+## Limitations
+
+* Because the initial LLDP data are in a non-uniform format, they will be mapped
+  to a uniform format.
+
+* There is a delay from receiving the LLDP data on the oVirt host, propagating
+  the data to the oVirt engine and presenting the data to the user.
+
+## Benefit to oVirt
+
+Information from LLDP provided by oVirt engin's REST-API will support the
+integration of oVirt in other software.
+
+Information from LLDP provided by the UI might help the user to avoid and
+locate manual host network configuration errors caused by a mismatch of switch
+port configuration.
+
+The engine could block manual misconfiguration and provide a smart assistance
+during host network configuration of VLANs, bonds and MTU.
+
+## Requirements to the first implementation iteration
+
+* The TLVs:
+  * Chassis ID TLV (Type = 1)
+  * Port ID TLV (Type = 2)
+  * Port Description TLV (Type = 4), with a value of up to 2000 chars
+  * System Name TLV (Type = 5)
+  * System Description TLV (Type = 6)
+  * Management Address TLV (Type = 8)
+  has to be available on the engine's REST API
+
+* Is there an upper limit of the acceptable age of the data?
+
+## Entity Description
+
+The initial LLDP information is a list of type, length, value (TLV) tuples.
+To express multiple values, e.g. to list the allowed VLANs on a switch port,
+the TLV is sent multiple times.
+The value may contain structured data.
+This way the LLDP data is non-uniform.
+This non-uniform LLDP data will be mapped to the following uniform entities.
+
+### tlv
+
+A `tlv` is associated to a `vds_interface`, has a `name`, a `description` and a
+`value`.
+
+The `description` is a human readable string to describe what the value is about.
+
+The `name` is a string created by rules, to allow a bidirectional mapping
+between the name and a tuple of `type`, `oui` and `subtype` of the LLDP's
+original TLV. This would result in names similar the ones used in OpenStack [4].
+
+The `value` may be a single simple string, or an ordered list of `tlv_value`s.
+
+There will be a new table in engine to store the `tlv`s. The table needs columns
+for `id`, `vds_interface_id`, `name`, `description` and `value`.
```

Comment:
I do not think a tlv object should have any idea about an interface object, it is the other way around.